An error occured at 2023-12-30 21:40:04:

function: wrapper
args: ({'site_data': {'prompt': "For the sites ${products}\nGenerate the following commands and give me every command again with its answer:\n1-Give me a good hook title for the site comparing those products, and include the names of the products if they're two or less\n2-give me a good seo description for the site\n3-Give me the best practice keywords to include in this site for seo as a python list", 'extracted_data': ['site_title', 'site_description', 'site_keywords']}, 'site_introduction': {'prompt': 'For a site that is comparing between the products ${products} generate a good introduction optimized for seo, and avoid including all the products names in it', 'extracted_data': ['site_introduction']}, 'site_conclusion': {'prompt': 'For a site that is comparing between the products ${products} generate a good conclusion optimized for seo, and avoid including all the products names in it', 'extracted_data': ['site_conclusion']}}, <google.generativeai.generative_models.ChatSession object at 0x000002986E504950>, ['${products}'], ["['Adidas', 'Nike']"])
kwargs: {}

Traceback (most recent call last):
  File "C:\Users\ideapad\AppData\Local\Programs\Python\Python312\Lib\site-packages\pam.py", line 412, in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ideapad\AppData\Local\Programs\Python\Python312\Lib\site-packages\pam.py", line 399, in wrapper
    match input("Are you happy with the results? y\\n\n").lower():
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt



